### DISCUSSION TITLE
URL https://uks1.rel.tunnels.api.visualstudio.com is categorized as "Proxy Avoidance" by FortiGuard Intrusion Prevention

### DISCUSSION THREAD

**Discussion by farnulfo** [community/community#134171] [Codespaces]
### Select Topic Area

General

### Body

Hi,
This url https://uks1.rel.tunnels.api.visualstudio.com is used in Github Codespace and is blocked by FortiGuard because it's catagozied as "Proxy Avoidance" and in our work environment this category is blocked.
I've already sent request to change category for this url but it's not working.

Can you contact and deal with Fortinet / FortyGuard to change this category ?
![image](https://github.com/user-attachments/assets/ecb0f744-8d3a-4976-87c5-90a18029d16c)

**Comment by Akash1134** [comment]
Hi @farnulfo ,

Thanks for reporting. Have you discussed this issue with your IT department to see if there are any temporary workarounds they can provide, such as whitelisting the specific URL?

Thanks!

**Comment by philippe-granet** [comment]
Same problem, URL `https://uks1.rel.tunnels.api.visualstudio.com/tunnels/.../ports/16635?api-version=2023-09-27-preview` is blocked (by our corporate proxy or Crowdstrike Agent, I don't know) and we can't access to Github Codespace
